fractal_server/app/schemas/_validators.py

```diff
@@ -63,3 +63,13 @@ def val(string: str):
         return s
 
     return val
+
+
+def val_unique_list(attribute: str):
+    def val(must_be_unique: list):
+        if must_be_unique is not None:
+            if len(set(must_be_unique)) != len(must_be_unique):
+                raise ValueError("`slurm_accounts` list has repetitions")
```

Comment:
If the error message includes "slurm_accounts", then let's turn this into a standard validator (included in the Pydantic models, rather than re-usable). Or let's change the name to something like `unique_slurm_accounts_list`.